src/webserver/HttpBase.h

```diff
@@ -380,12 +401,6 @@ handle_request(
         return send(httpResponse(
             http::status::bad_request, "text/html", "Expected a POST request"));
 
-    if (!dosGuard.isOk(ip))
```

Comment:
remove dosGuard in handle_request. the check should happen outside the business logic.